pyunlocbox/functions.py

```diff
@@ -867,19 +858,78 @@ def _prox(self, x, T):
         return np.clip(x, 0, np.inf)
 
 
+class proj_spsd(proj):
+    r"""
+    Projection on symmetric positive semi-definite matrices (eval, prox).
+
+    This function is the indicator function :math:`i_S(M)` of the set
+    :math:`S = \left\{M \in \mathbb{R}^{N \times N}
```

Comment:
I have to make two lines here otherwise I had too many characters. Is it ok?

Reply:
yep thanks